mlrun/db/httpdb.py

```diff
@@ -1035,6 +1035,8 @@ def paginated_list_runs(
 
         :param page: The page number to retrieve. If not provided, the next page will be retrieved.
         :param page_size: The number of items per page to retrieve. Up to `page_size` responses are expected.
+            The default value is specified in `mlrun.mlconf.httpdb.pagination.default_page_size`. If not provided,
+            the default page size will be used.
```

Comment:
Can be shorter and clearer :
```suggestion
            Defaults to `mlrun.mlconf.httpdb.pagination.default_page_size` if not provided.
```